.github/workflows/test.yml

```diff
@@ -21,4 +21,4 @@ jobs:
     - name: Check code style
       run: |
         pip install ruff
-        ruff format --check --diff .
+        ruff check .
```

Comment:
The lint needs to be its own step - it does not include the format checks. IMO it should be a separate step, so we'd have `Check code formatting` and `Check for code lint` or something